BMM.Core/Implementations/DeepLinking/Base/DeepLinkParserBase.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using System.Threading.Tasks;
+
+namespace BMM.Core.Implementations.DeepLinking.Base
+{
+    public abstract class DeepLinkParserBase
```

Comment:
Hm. Not sure about this one.
I will need to look at the rest and will come back to this.